katas/langtonant/Giovanni/langtonsant.js

```diff
@@ -0,0 +1,61 @@
+const mapdim = 6;
```

Comment:
In order to follow convention, please use lower case for folder with your name `giovanni`